docs/src/lib/types.md

```diff
@@ -16,7 +16,31 @@ It is not intended as a fully generic interface for working with tabular data, w
 interfaces defined by [Tables.jl](https://github.com/JuliaData/Tables.jl/) instead.
 
 `DataFrame` is the most fundamental subtype of `AbstractDataFrame`, which stores a set of columns
-as `AbstractVector` objects.
+as `AbstractVector` objects. When a `DataFrame` is constructed columns *are not* copied, if possible.
+The exception is `DataFrame(::DataFrame)` constructor which performs a copy of the columns.
```

Comment:
```suggestion
The exception is `DataFrame(::AbstractDataFrame)` constructor which performs a copy of the columns.
```

(I am  guessing)

Reply:
We do not have such a constructor currently .

Reply:
We have `DataFrame(::SubDataFrame)` though right?

Reply:
Yes - but there it is not possible to run this constructor without copying that is why I have not included it (but now having thought about it I will add `SubDataFrame` for clarity - thanks).